classes/order/Order.php

```diff
@@ -691,6 +692,21 @@ protected function setProductCustomizedDatas(&$product, $customized_datas)
         }
     }
 
+    /**
+     * If order line targets a combination, replace row stock location data
+     * by combination stock location data.
+     *
+     * @param array &$product
+     */
+    protected function setProductStockLocationForCombination(&$product)
```

Comment:
wouldn't it rather be `setProductStockLocationFromCombination`

Reply:
Nice one 😉 I'll modify it